src/babelPluginTsdx.ts

```diff
@@ -10,7 +10,10 @@ export const isTruthy = (obj?: any) => {
   return obj.constructor !== Object || Object.keys(obj).length > 0;
 };
 
-const replacements = [{ original: 'lodash', replacement: 'lodash-es' }];
+const replacements = [
+  { original: 'lodash', replacement: 'lodash-es' },
+  { original: 'lodash-es/fp', replacement: 'lodash/fp' },
+];
```

Comment:
`babel-plugin-transform-rename-import` [supports RegEx](https://github.com/laat/babel-plugin-transform-rename-import#babelrc) -- I think using [a negative lookahead](https://stackoverflow.com/a/9594067/3431180) would be a cleaner fix for this

```suggestion
// replace lodash with lodash-es, but not lodash/fp
const replacements = [{ original: 'lodash(?!\/fp)', replacement: 'lodash-es' }];
```